tests/foreman/api/test_contentviewversion.py

```diff
@@ -11,33 +11,170 @@
 from robottelo.test import APITestCase
 
 
-class CVVersionTestCase(APITestCase):
-    """Tests for content view versions."""
+class ContentViewVersionCreateTestCase(APITestCase):
+    """Tests for content view version creation."""
 
-    def test_negative_promote_1(self):
-        """@Test: Promote the default content view version.
+    @classmethod
+    def setUpClass(cls):
+        """Single organization for all tests"""
+        super(ContentViewVersionCreateTestCase, cls).setUpClass()
+        cls.org = entities.Organization().create()
 
-        @Assert: The promotion fails.
+    def setUp(self):
+        """Init content view with repo per each test"""
+        super(ContentViewVersionCreateTestCase, self).setUp()
+        self.content_view = entities.ContentView(
+            organization=self.org,
+        ).create()
 
-        @Feature: ContentViewVersion
+    def test_positive_create(self):
+        """@Test: Create a content view version.
+
+        @Assert: Content View Version is created.
+
+        @Feature: Content View Version
+        """
+        # Fetch content view for latest information
+        cv = self.content_view.read()
+        # No versions should be available yet
+        self.assertEqual(len(cv.version), 0)
+
+        # Publish existing content view
+        cv.publish()
+        # Fetch it again
+        cv = cv.read()
+        self.assertGreater(len(cv.version), 0)
+
+    def test_negative_create(self):
+        """@Test: Create content view version using the 'Default Content View'.
+        set.
 
+        @Assert: Content View Version is not created
+
+        @Feature: Content View Version
         """
-        env = entities.Environment().create()
+        # The default content view cannot be published
+        cv = entities.ContentView(
+            organization=self.org,
+            name=u'Default Organization View'
```

Comment:
Are you suggesting I create a constant for it in this module? I couldn't find one anywhere...